oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java

```diff
@@ -1113,12 +1105,10 @@ private String mapActionStatus(String status) {
         } else if (CoordinatorAction.Status.WAITING.toString().equals(status)
             || CoordinatorAction.Status.SUBMITTED.toString().equals(status)) {
             return InstancesResult.WorkflowStatus.WAITING.name();
-        } else if (CoordinatorAction.Status.KILLED.toString().equals(status)) {
-            return InstancesResult.WorkflowStatus.KILLED.name();
         } else if (CoordinatorAction.Status.IGNORED.toString().equals(status)) {
-            return InstancesResult.WorkflowStatus.KILLED_OR_IGNORED.name();
+            return InstancesResult.WorkflowStatus.KILLED.name();
```

Comment:
This change is reversing a patch to ensure manually killed instances aren't retried.